message/message.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package message;
+
+enum MessageType {
+    JOIN_GAME = 0;
+    LEAVE_GAME = 1;
```

Comment:
Probably this should be served via EPIPE signal from unsuccessfull socket write. This should be helpful, because we'll code only one unified case instead of handling LEAVE_GAME message and EPIPE.